### ISSUE TITLE
'ValidTransientAggregatorIdentifiers' cannot package

### ISSUE THREAD

**Issue by Chriscross475** [tranek/GASDocumentation#73]
When I try to package my game, I get the following error when using 'ValidTransientAggregatorIdentifiers':

`error C2065: 'ValidTransientAggregatorIdentifiers': undeclared identifier`

Everything works fine in the editor. 

In VS I also can only build with "...Editor" solution config without getting errors,

**Comment by tranek** [comment]
Interesting find. I've not personally used them beyond simple testing to write about them in the doc. What are the ramifications of that preprocessor macro? Does it not work on dedicated servers where the execution calculation would run?

**Comment by Chriscross475** [comment]
Yes, they are working just fine in the editor, that makes it hard to find. I am not sure about that ramifications of that preprocessor macro and dedicated servers.
But I just added the `GameplayTagContainer ValidTransientAggregatorIdentifiers` again on my own child class and now its recognized also when packaging. Not sure if this adds other errrors because I simple dublicated the variable, so not closing the issue yet. I'll try some more testing, but it sems to work for now.

**Comment by Chriscross475** [comment]
Okay its confusing as hell, when I added ValidTransientAggregatorIdentifiers on my child, all seems to work, but the execution does not recognize the temporal data as attributes anymore. I just figured when I wanted to add another Damage Gameplay Effect. I cannot choose those anymore and they got replaced in all created before.
But somehow it still gets the Damage data what does not make any sense at all.

**Comment by Chriscross475** [comment]
okay i asked in the Unreal Slackers Discord and somebody pointed out you need to add the #if WITH_EDITORONLY_DATA ...#endif also to your execution like this for example:

#if WITH_EDITORONLY_DATA
	ValidTransientAggregatorIdentifiers.AddTag(FGameplayTag::RequestGameplayTag("Data.Damage"));
#endif

I think you should add this in the documentation
